governance.md

```diff
@@ -84,7 +86,7 @@ Current TOC members:
 * Rob Zienert (Netflix)
 * Travis Tomsu (Google)
 
-The technical oversight committee is initially seeded with core members from Netflix and Google. We will leave further details such as number of seats, qualifications, tenure, TBD for the time being.
+The TOC should remain with an odd number of members (between 3 to 7) so as to avoid deadlock. In the event that someone from TOC leaves prematurely, the SC should be proactive in filling the spot to ensure an odd number of members. The TOC will nominate a chair person for a 1 year term. Their duty is to run/call meetings, take meeting notes, and call votes.  
```

Comment:
[minor] To avoid potential ambiguity about whether "between" does or does not include the boundaries, perhaps phrase simply as "...should remain with 3, 5 or 7 members so..."?

Reply:
I would suggest replacing the new text with the following:

The TOC will have three, five, or seven members, as deemed appropriate by the Steering Committee.  In the event that someone from the TOC leaves during their term, the Steering Committee shall appoint someone to fill the remainder of the term.  Each year, the TOC will nominate a member to serve as the Chairperson for that year.  The Chair's duties will include calling and runnings meetings, calling votes, and ensuring that meeting notes are recorded.